documentation/source/roadmap.rst

```diff
@@ -3,9 +3,13 @@ Roadmap
 *******
 
 cocotb is in active development.
+Our release cycle is about twice per year.
```

Comment:
I'm not thrilled about the use of possessive modifiers in this section of the documentation.

> cocotb is in active development

I'm not sure if this is correct. You might not agree depending on your interpretation of "active development". It might be better to spell out how our development looks like, like you started to do by mentioning the release cycle.

```suggestion
cocotb is under development; however there is no "master plan".
We depend upon users to contribute new features and improvements while maintainers focus mostly on reviewing PRs, project management, and bugfixes.
Releases will occur approximately every 6 months.
```

Reply:
I also think the "Roadmap" title to this section is a bit misleading. Maybe "Development and Contributing"? Maybe we can link the CONTRIBUTING.md and/or contribution documents once they are actually written (#2057).

Reply:
And there I go using possessive modifiers 😆. Maybe it's not a big deal.

Reply:
I used your wording suggestion; renaming the section and merging some other content is a TODO comment in ``index.rst`` already.